src/importer.rs

```diff
@@ -680,19 +680,45 @@ mod tests {
 
     #[test]
     fn test_importer_import_obj_cache_returns_the_same_cached_data_from_different_file() {
+        lazy_static! {
+            static ref MODELS: Vec<Model> = vec![Model {
+                name: "test".to_string(),
+                geometry: Geometry::from_triangle_faces_with_vertices_and_normals(
+                    vec![TriangleFace::new(0, 1, 2)],
+                    vec![
+                        Point3::new(6.0, 5.0, 4.0),
+                        Point3::new(3.0, 2.0, 1.0),
+                        Point3::new(0.0, 1.0, 2.0),
+                    ],
+                    vec![
+                        Vector3::new(1.0, 0.0, 0.0),
+                        Vector3::new(1.0, 0.0, 0.0),
+                        Vector3::new(1.0, 0.0, 0.0),
+                    ],
+                ),
+            }];
+        }
+        let path = "tests/fixtures/valid.obj";
         let mut cache = MockObjCache::new();
         cache
             .expect_get_if_not_modified()
             .returning(|_, _| None)
             .times(1);
         cache
             .expect_get_by_checksum()
-            .returning(|_| Some(vec![]))
+            .returning(|_| Some(MODELS.to_vec()))
+            .times(1);
+        cache
+            .expect_set()
+            .with(
+                predicate::eq(path.to_string()),
+                predicate::always(),
```

Comment:
Should this `always` be here? Also, are these predicates AND-ed or OR-ed>

Reply:
These are 3 different parameters, so nothing is OR-ed or AND-ed here.

`always` was there just so we don't need to deal with file metadata, but it's probably better if we do this thoroughly. I've changed it.